app/views/twilio/call.haml

```diff
@@ -1,3 +1,3 @@
 !!! XML
-%Response
-  %Dial= @user.phone
+%response
```

Comment:
Haml lint doesn't seem to notice the xml extension and still points out the "issues".